ckan/controllers/group.py

```diff
@@ -608,17 +637,22 @@ def delete(self, id):
                 self._action('group_delete')(context, {'id': id})
                 if self.group_type == 'organization':
                     h.flash_notice(_('Organization has been deleted.'))
-                else:
+                elif self.group_type == 'group':
                     h.flash_notice(_('Group has been deleted.'))
-                self._redirect_to(controller='group', action='index')
+                else:
+                    h.flash_notice(_('%s has been deleted.')
+                                   % _(group_type.capitalize()))
```

Comment:
I've put back the original flash messages, to avoid disrupting the default group_types. And although it is a hidden string here for a custom group_type, it is translatable if people spot it. Am not sure what else we can do.